cocos/gfx/base/pipeline-state.editor.ts

```diff
@@ -319,7 +319,7 @@ export class PassStatesEditor implements EffectAsset.IPassStates {
     @type(BlendStateEditor)
     @serializable
     @editable
-    public blendState: BlendStateEditor = new BlendStateEditor();
+    public blendState: any = new BlendStateEditor();
```

Comment:
Why change to `any`?

Reply:
The definition here is incorrect, and I don't know how to solve it. The base type is expecting a BlendState, but a BlendStateEditor is given here. In fact, there have been errors in this code for a long time.